src/types/orchestrationTypes.ts

```diff
@@ -65,3 +65,224 @@ export interface CallHttpOptions {
      */
     asynchronousPatternEnabled?: boolean;
 }
+
+/**
+ * Provides functionality for application code implementing an orchestration operation.
+ */
+export declare class DurableOrchestrationContext {
```

Comment:
Same feedback as https://github.com/Azure/azure-functions-durable-js/pull/450#discussion_r1100574673

This comment is the only reason I'm requesting changes

Reply:
This is the type signature of the constructor:

```TS
constructor(
        state: HistoryEvent[],
        instanceId: string,
        currentUtcDateTime: Date,
        isReplaying: boolean,
        parentInstanceId: string | undefined,
        longRunningTimerIntervalDuration: string | undefined,
        maximumShortTimerDuration: string | undefined,
        defaultHttpAsyncRequestSleepTimeMillseconds: number | undefined,
        schemaVersion: ReplaySchema,
        input: unknown,
        private taskOrchestratorExecutor: TaskOrchestrationExecutor
    );
```

Exposing the `constructor()` would also force us to expose other types like `HistoryEvent` (which in turn exposes `HistoryEventType`), `ReplaySchema`, `TaskOrchestrationExecutor` (which in turn also exposes `TaskBase` and other internal classes), etc. In the interest of #352, and in the spirit of this change to let us more intentionally the public APIs, I think we should omit the constructor. Users shouldn't be using this constructor anyway.

Reply:
Added constructor, will file issue to remove